package.json

```
@@ -7,13 +7,15 @@
     "start": "node ./src/bin/www",
     "devstart": "babel-watch ./src/index.js",
     "build": "rm -rf dist && mkdir dist && babel src -s -d dist",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha -r @babel/register src/tests/*.js --exit"
```

```suggestion
    "test": "NODE_ENV=test&& mocha -r @babel/register src/tests/*.js --exit"
```
This is to update our environment to test when running tests so as to pull in the right config variables